src/Symmetric/PaddingPluginManager.php

```diff
@@ -45,7 +46,12 @@ public function has($id)
      */
     public function get($id)
     {
+        if (! $this->has($id)) {
+            throw new NotFoundException(
+                'The padding adapter %s does not exist', $id
```

Comment:
You're using an `sprintf()` notation, without the `sprintf()` wrapping it. :smile: